### PR TITLE
[PHP 8.4] Add errors for GetText functions

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: reference/gettext/functions/dgettext.xml [hunk 1]
```diff
@@ -49,7 +49,38 @@
    A <type>string</type> on success.
   </para>
  </refsect1>
- 
+
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <simpara>
+   Throws a <classname>ValueError</classname> if <parameter>domain</parameter>
```
Comment (Girgias) [inline reference/gettext/functions/dgettext.xml:56]: Should be `<exceptionname>`

**Review by Girgias — CHANGES_REQUESTED** [review]
Tag should be amended

File: reference/gettext/functions/bind-textdomain-codeset.xml [hunk 2]
```diff
@@ -52,6 +52,14 @@
   </para>
  </refsect1>
 
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <simpara>
+   Throws a <exceptionname>ValueError</exceptionname> if <parameter>domain</parameter>
+   is the empty &string;.
```
Comment (cmb69) [inline reference/gettext/functions/bind-textdomain-codeset.xml:59]: As I understand it, nowaday the following is preferred:
```suggestion
   is the empty <type>string</type>.
```

Comment (saundefined) [reply reference/gettext/functions/bind-textdomain-codeset.xml]: I missed that, thanks! I'll keep that in mind

**Comment by Girgias** [comment]
Yes, except if we want to wait for the PR that uses PI to land and use those instead?

**Pull request merged**